services/storage-users/pkg/command/trash_bin.go

```diff
@@ -1,23 +1,59 @@
 package command
 
 import (
+	"context"
+	"fmt"
+	"path"
+	"path/filepath"
+	"strings"
 	"time"
 
+	gateway "github.com/cs3org/go-cs3apis/cs3/gateway/v1beta1"
+	rpc "github.com/cs3org/go-cs3apis/cs3/rpc/v1beta1"
+	provider "github.com/cs3org/go-cs3apis/cs3/storage/provider/v1beta1"
 	"github.com/cs3org/reva/v2/pkg/events"
+	"github.com/cs3org/reva/v2/pkg/rgrpc/todo/pool"
+	"github.com/cs3org/reva/v2/pkg/storagespace"
+	"github.com/cs3org/reva/v2/pkg/utils"
+	"github.com/mohae/deepcopy"
 	"github.com/owncloud/ocis/v2/ocis-pkg/config/configlog"
+	"github.com/owncloud/ocis/v2/ocis-pkg/tracing"
 	"github.com/owncloud/ocis/v2/services/storage-users/pkg/config"
 	"github.com/owncloud/ocis/v2/services/storage-users/pkg/config/parser"
 	"github.com/owncloud/ocis/v2/services/storage-users/pkg/event"
+	"github.com/owncloud/ocis/v2/services/storage-users/pkg/logging"
+	"github.com/rs/zerolog/log"
 	"github.com/urfave/cli/v2"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
 )
 
+const (
+	SKIP = iota
+	REPLACE
+	KEEP_BOTH
+
+	retrievingErrorMsg = "trash-bin items retrieving error"
```

Comment:
```suggestion
	_retrievingErrorMsg = "trash-bin items retrieving error"
```

Please prefix unexported global consts with `_`. That makes it easier to spot them.